Readme.md

```diff
@@ -68,6 +68,8 @@ $ sudo make install
 
 ## Examples
 
+ For full example/best practice, click [here](https://github.com/clibs/clib/blob/master/BEST_PRACTICE.md).
```

Comment:
Maybe reword this as:

> More examples and best practices at BEST_PRACTICE.md

IMO it reads a bit better and [doesn't have a "click here" link](https://webaccess.berkeley.edu/ask-pecan/click-here).